README.md

```diff
@@ -17,27 +17,27 @@ You can import the script as a module by adding the repository to your path in a
 ```python
 import sys
 sys.path.append('/path/to/intro_machine_learning_using_python')
-import script as s
+import vendor_data as vd

```

Comment:
Note that the `as vd` part is not _necessary_, this was just an example. But if you want to use it like that, go ahead :)